File: milestone 2/python_modules/module00/ex1/ft_plot_area.cpp
#include <iostream>

void	ft_plot_area()
{
	int length, width, area;
	std::cout << "Enter length: ";
	std::cin >> length;
	std::cout << "Enter width: ";
	std::cin >> width;
	area = width * length;
	std::cout << "Plot area: " << area;
	std::cout << std::endl;
}

int		main(void)
{
	ft_plot_area();
	return 0;
}

File: milestone 2/python_modules/module00/ex1/ft_plot_area.java
import java.util.Scanner;

public class ft_plot_area {
    public static void main(String[] args) {
        System.out.print("Enter length: " );
        Scanner scan = new Scanner(System.in);
        int width = scan.nextInt();
        System.out.print("Enter width: ");
        Scanner scan2 = new Scanner(System.in);
        int height = scan2.nextInt();
        System.out.println("Plot area: " + (width * height));
    }
}